### PR TITLE
Assorted tweaks and updates

### PR DESCRIPTION
**Pull request by XhmikosR**
@bryanbraun Please don't squash this and don't let it hanging for a long time :)

Let me know if you need me to make any changes.

Notes:

* I didn't update the dist files, etc, but after merging this, a new patch release would be perfect.
* Ideally, we should have karma run for the minified file just to be 100% safe, but that's for another time (we should have an issue to track it)
* We could add coverage support later

Manual Preview: <https://sleepy-noyce-bed069.netlify.app>

### CODE REVIEW

File: docs/scripts.js [hunk 1]
```diff
@@ -1,31 +1,35 @@
...
 }
 
 document.addEventListener('DOMContentLoaded', function() {
-  var toggleButtons = document.getElementsByClassName('example-code-link'); // get the buttons
+  // get the buttons
+  var toggleButtons = document.querySelectorAll('.example-code-link');
 
-  for(var i = 0; i < toggleButtons.length; i++) {
-    toggleButtons[i].addEventListener("click", slideExampleCode); // add the event to each button
+  for (var i = 0; i < toggleButtons.length; i++) {
```
Comment (XhmikosR) [inline docs/scripts.js:20]: I could probably use one variable, but I went with the way I'd go if this was a let variable. Feel free to change it later :)

File: test/visual-check.html [hunk 2]
```diff
@@ -39,7 +39,6 @@
 </head>
 <body>
   <section class="main">
-    <a href="https://github.com/bryanbraun/anchorjs"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://github-camo.global.ssl.fastly.net/365986a132ccd6a44c23a9169022c0b5c890c387/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f7265645f6161303030302e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_red_aa0000.png"></a>
     <h1 class="page-title">AnchorJS</h1>
     <div class="anchorjs-logo"></div>
```
Comment (XhmikosR) [inline test/visual-check.html:43]: We could probably remove this too or use the SVG file, but I thought I'd leave this to you :)

File: test/load-order-3.html [hunk 3]
```diff
@@ -32,7 +32,7 @@
 <body>
   <section class="main">
     <h1>A page with large images (delays after DOMContentLoaded).</h1>
-    <img src="http://lorempixel.com/1600/1600/" alt="Large image">
+    <img src="https://loremflickr.com/1600/1600/" alt="Large image">
```
Comment (XhmikosR) [inline test/load-order-3.html:35]: I don't get a 1600x1600 image, though, but I don't have a replacement in mind. Feel free to adapt this later :)

File: anchor.js [hunk 4]
```diff
@@ -244,8 +244,7 @@
 
       // Regex for finding the non-safe URL characters (many need escaping):
       //   & +$,:;=?@"#{}|^~[`%!'<>]./()*\ (newlines, tabs, backspace, vertical tabs, and non-breaking space)
-      var nonsafeChars = /[& +$,:;=?@"#{}|^~[`%!'<>\]./()*\\\n\t\b\v\u00A0]/g,
-          urlText;
```
Comment (XhmikosR) [inline anchor.js:248]: I can revert the temp var removal; I thought it helped with uglify but apparently it doesn't. So, if it's more clear for you with the temp var, let me know and I'll revert it.

File: docs/index.html [hunk 5]
```diff
@@ -19,7 +19,7 @@
   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:200,400,700&display=swap">
   <link rel="stylesheet" href="styles.css">
   <link rel="stylesheet" href="fonts/fonts.css">
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/prismjs@1.21.0/themes/prism-twilight.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/prismjs@1.23.0/themes/prism-twilight.css">
```
Comment (XhmikosR) [inline docs/index.html:22]: This update fixes IE 11 errors among other things

File: anchor.js (hunk 4 above)
Comment (bryanbraun) [reply anchor.js:248]: I think it's good without the temp variable. 👍

File: test/load-order-3.html (hunk 3 above)
Comment (bryanbraun) [reply test/load-order-3.html:35]: Gotcha. This is definitely an improvement.

**Review by bryanbraun — APPROVED** [review]
These are really great changes all around! Thanks for your patience waiting until I could get to it. I'll merge it and roll out a patch update shortly. 👍

File: test/load-order-3.html (hunk 3 above)
Comment (alpadev) [reply test/load-order-3.html:35]: How about https://picsum.photos/1600 ? 😊

**Pull request merged**